modoptions.lua

```diff
@@ -268,7 +274,7 @@ local options = {
 		type	= "bool",
 		section	= "options_main",
 		def		=  false,
-		column	= 1.76,
+		column	= 2.33,
```

Comment:
What is this change for?

Reply:
This is handled where? Chobby? Settings Panel?

Reply:
Yes

Reply:
Ok, I'll just assume this change makes sense.